inc/SimpleExpressionCalculatorW3.h

```diff
@@ -0,0 +1,51 @@
+///////////////////////////////////////////////////////////////////////////////
+/**
+ * @file  SimpleExpressionCalculatorW3.h
+ * @date  Wed, 2 June 2021
+ * @brief 
+ * 
+ */
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef SIMPLEEXPRESSIONCALCULATORW3_H
+#define SIMPLEEXPRESSIONCALCULATORW3_H
+
+#include <string>
+
+#include "ICalculatorW3.h"
+
+namespace calculator 
+{
+    class SimpleExpressionCalculatorW3 : public ICalculatorW3
+    {
+        public:
+            /**
+             * @brief The left side of the equation [leftTerm] [operation] [rightTerm],  e.g. '5 + 5'
+             * 
+             * @return std::string 
+             */
+            std::string getExpression() override final;
+            /**
+             * @brief The solved right side of the equation
+             * 
+             * @return int 
+             */
+            virtual int getResult() = 0;
+            /**
+             * @brief The full equation represented as a string [leftTerm] [operation] [rightTerm] = [result]
+             * 
+             * @return std::string 
+             */
+            std::string toString() override final;
+        protected:
```

Comment:
why protected instead of private?

Reply:
this is a base class for [Add/Subtract/Divide/Multiply]Calculator. Doesn't it need to be protected and not private if I want all of the derived classes to have those members?